workflows/mag.nf

```diff
@@ -96,10 +94,13 @@ include { BINNING_PREPARATION } from '../subworkflows/local/binning_preparation'
 include { BINNING             } from '../subworkflows/local/binning'
 include { BINNING_REFINEMENT  } from '../subworkflows/local/binning_refinement'
 include { BUSCO_QC            } from '../subworkflows/local/busco_qc'
+include { VIRUS_IDENTIFICATION} from '../subworkflows/local/virus_identification'
 include { CHECKM_QC           } from '../subworkflows/local/checkm_qc'
 include { GUNC_QC             } from '../subworkflows/local/gunc_qc'
 include { GTDBTK              } from '../subworkflows/local/gtdbtk'
 include { ANCIENT_DNA_ASSEMBLY_VALIDATION } from '../subworkflows/local/ancient_dna'
+include { DOMAIN_CLASSIFICATION } from '../subworkflows/local/domain_classification'
+include { DEPTHS              } from '../subworkflows/local/depths'
```

Comment:
Curly brackets are not aligned here (and I am only pointing it out since they are in alignment everywhere else 😆)

Reply:
Done!